app/ledger.js

```diff
@@ -111,6 +111,18 @@ const doAction = (action) => {
       if (action.key === settings.PAYMENTS_CONTRIBUTION_AMOUNT) return setPaymentInfo(action.value)
       break
 
+    case appConstants.APP_CHANGE_SITE_SETTING:
+      if (action.key === 'ledgerPaymentsShown') {
+        // TODO
```

Comment:
@mrose17 if `action.value === false` then the user has right-clicked to delete a ledger synopsis entry